ui/app/AppLayouts/Profile/views/AppearanceView.qml

```diff
@@ -199,36 +160,24 @@ SettingsContentBase {
                 Layout.preferredHeight: implicitHeight
                 image.source: Style.png("appearance-light")
                 control.text: qsTr("Light")
-                control.checked: localAppSettings.theme === Universal.Light
-                onRadioCheckedChanged: {
-                    if (checked) {
-                        appearanceView.updateTheme(Universal.Light)
-                    }
-                }
+                control.checked: currentTheme === 0
+                onRadioCheckedChanged: { themeChanged(0) }
             }
 
             StatusImageRadioButton {
                 Layout.preferredWidth: parent.width/3 - parent.spacing
                 image.source: Style.png("appearance-dark")
                 control.text: qsTr("Dark")
-                control.checked: localAppSettings.theme === Universal.Dark
-                onRadioCheckedChanged: {
-                    if (checked) {
-                        appearanceView.updateTheme(Universal.Dark)
-                    }
-                }
+                control.checked: currentTheme === 1
```

Comment:
Why are we suddenly referring to the current theme via some magic constant instead of `Universal.Dark`? (same for all the radio buttons)